tests/contracts/bit-xor.clar

```diff
@@ -0,0 +1,2 @@
+(define-public (assert)
+  (ok (bit-xor 3 2)))
```

Comment:
Please add another test using `xor`.

Reply:
I looked at that but it failed, i would prefer keeping the scope of this pr to the bitwise fns so it can finally land.